It breaks existing ts configuration

Because `@types/node` is specified as a dependency, it breaks my existing project configuration: 
```
"@types/node": "12.12.12"
"typescript": "3.7.2"
```
Error:
> node_modules/@types/node/index.d.ts:200:11 - error TS2300: Duplicate identifier 'IteratorResult'.

The solution would be to move `@types/node` under `devDependencies`.

--- a/package.json
+++ b/package.json
@@ -49,7 +49,6 @@
     "types"
   ],
   "dependencies": {
-    "@types/node": "12.7.5",
     "bech32": "^1.1.2",
     "bip174": "^1.0.1",
     "bip32": "^2.0.4",
@@ -69,6 +68,7 @@
   "devDependencies": {
     "@types/bs58": "^4.0.0",
     "@types/mocha": "^5.2.7",
+    "@types/node": "12.7.5",
     "@types/proxyquire": "^1.3.28",
     "bip39": "^3.0.2",
     "bip65": "^1.0.1",

